src/components/ContentBlock.tsx

```diff
@@ -7,8 +7,8 @@ interface ContentBlockProps {
   children: ReactNode;
 }
 
-const ContentBlock = ({ title, children }: ContentBlockProps) => (
-  <ContentBody>
+const ContentBlock = ({ title, children, background_color }: ContentBlockProps) => (
```

Comment:
ik it was already "background_color" but for consistency (camelCase) use "backgroundColor" (or maybe even "backgroundColour")